.eslintrc.js

```diff
@@ -63,6 +62,14 @@ module.exports = {
       plugins: ['node'],
       extends: ['plugin:node/recommended'],
     },
+    {
+      // test files
+      files: ['tests/**/*-test.{js,ts}'],
+      extends: [
+        // @todo enable this while fixing linting errors in tests...
```

Comment:
There were too many linting errors to fix in this PR. Should revisited this though!